### PR TITLE
Earth dot gov carousel

### PR DESCRIPTION
**Pull request by freitagb**
## Why are you creating this Pull Request?

With a global sea level portal we have been requested to add another link to highlight the new portal. Since we have 3 featured portals we requested integration of the carousel feature for Earth.gov. This PR configures the carousel as requested by HQ leadership

### CODE REVIEW

**Review by amarouane-ABDELHAK — APPROVED** [review]

File: overrides/home/carousel/items.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
     },
     {
-    link: '/stories/agriculture',
-    title: 'Internal link to the Agriculture story',
-    description: 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean vitae lorem et neque egestas mollis. Praesent ut porttitor mauris, a sagittis erat',
-    image: new URL('../media/hyperwall-image1.png', import.meta.url).href,
-    imageAlt: 'Example image alt EIC 2024'
-  }]
\ No newline at end of file
+    link: 'https://earth.gov/sealevel/us',
```
Comment (j08lue) [inline overrides/home/carousel/items.js:17]: This link to `https://earth.gov/sealevel/us` currently redirects me to `https://earth.gov/sealevel/us/us/`, which then gives a 404. It seems to work when I enter `https://earth.gov/sealevel/us/` (trailing slash) instead.

Is the `/us` needed, at all?

```suggestion
    link: 'https://earth.gov/sealevel/us/',
```

**Review by j08lue — CHANGES_REQUESTED** [review]
The carousel generally works, but a few things need fixing:

1. The link to sealevel does not work (see comment). A trailing slash seems to fix it.
2. There seem to be a bunch of very similar sealevel media images added - are they all needed?
3. The carousel needs to be better aligned with the title above, perhaps by using the grid differently. We might need to ping an engineer to help fix this - on first look, it is not obvious to me how to fix this.

   image

File: overrides/home/carousel/items.js (hunk 1 above)
Comment (freitagb) [reply overrides/home/carousel/items.js:17]: yeah - the /us i the federal portal and sealevel is the global. We've been asked to highlight both portals on earth.gov

**Comment by freitagb** [comment]
1.  I committed your change on this. Good catch
2. I don't believe there's a way to add text on the carousel card so the similar images are one with text for the carousel and one without - though we may not need the ones without text in the future. Will come back to this
3. Yeah - i was struggling with that too. I think it's okay on the landing page (will defer to designers) but the alignment does not look good on */stories/sea-level-rise page.

> The carousel generally works, but a few things need fixing:
> 
> 1. The link to sealevel does not work (see comment). A trailing slash seems to fix it.
> 2. There seem to be a bunch of very similar sealevel media images added - are they all needed?
> 3. The carousel needs to be better aligned with the title above it and other elements on the page, perhaps by using the grid differently. We might need to ping an engineer to help fix this - on first look, it is not obvious to me how.
>    image

File: stories/components/slr_carousel/items.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+export default [
+    {
+      link: 'https://earth.gov/sealevel',
+      title: 'Global Sea Level Rise',
+      description: 'The Global Sea Level Rise Portal is an international collaboration that offers essential information on current and project sea level rise to inform coastal communities across the globe.',
+      image: new URL('../media/global_slr.png', import.meta.url).href,
+      imageAlt: 'Coastline with crashing waves'
+    },
+    {
+    link: 'https://earth.gov/sealevel/us',
```
Comment (j08lue) [inline stories/components/slr_carousel/items.js:10]: Need trailing slash here, too

```suggestion
    link: 'https://earth.gov/sealevel/us/',
```

**Review by j08lue — APPROVED** [review]
Nice, the links now work and alignment is decent.

On the home page, it looks a bit busy with different width blocks. But unless you can make the Carousel a couple of columns wider to match the elements above, I think there is no better way of handling these different elements.

**Comment by j08lue** [comment]
Not sure the new (https://github.com/NASA-IMPACT/veda-config-eic/pull/149/commits/8f8f3f59b87b3810ade111a144148cb966b3a32b) alignment of the carousel title on the home page is better.

image

Not much to do, I guess?

**Comment by dzole0311** [comment]
I adjusted the width of the carousel grid so that it matches the rest of the grid for the page. This can be also configured on a per-instance basis using a USWDS config file to override the default width / spacings of the grid (Docs: https://designsystem.digital.gov/documentation/settings/#configuring-custom-uswds-settings)

Hope this works?

**Comment by j08lue** [comment]
Nice! The carousel is massive now - could we also make both the carousel and its title have the same width as the elements above?

**Comment by dzole0311** [comment]
@j08lue Okay, I replaced some of the legacy grid styles with the USWDS grid. There might be some minor differences in the width of the grid from before (e.g the hero description might have been 50px wider before due to using legacy grid), but otherwise it should be okay. If anything, it can be easily tweaked now using some of the USWDS classes for spacing: https://designsystem.digital.gov/utilities/margin-and-padding/

**Pull request merged**